src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmBatchProcessResponseFilesImpl.java

```diff
@@ -0,0 +1,646 @@
+                        log.error(UNABLE_TO_UPDATE_EOD, e);
+                    }
+                }
+            }
+        );
+    }
+
+    private void processCreateRecordResponseFiles(List<CreateRecordFilenameProcessor> createRecordResponses,
+                                                  ArmBatchResponses armBatchResponses) {
+        for (CreateRecordFilenameProcessor createRecordFilenameProcessor : createRecordResponses) {
+            try {
+                BinaryData createRecordBinary = armDataManagementApi.getBlobData(createRecordFilenameProcessor.getCreateRecordFilename());
+                readCreateRecordFile(createRecordBinary, createRecordFilenameProcessor, armBatchResponses);
+            } catch (Exception e) {
+                log.error("Unable to process upload file {}", createRecordFilenameProcessor.getCreateRecordFilename());
+            }
+        }
+    }
+
+    private void readCreateRecordFile(BinaryData createRecordBinary, CreateRecordFilenameProcessor createRecordFilenameProcessor,
+                                      ArmBatchResponses armBatchResponses) {
+        if (nonNull(createRecordBinary)) {
+            Path jsonPath = null;
+            try {
+                boolean appendUuidToWorkspace = true;
+                jsonPath = fileOperationService.saveBinaryDataToSpecifiedWorkspace(
+                    createRecordBinary,
+                    createRecordFilenameProcessor.getCreateRecordFilename(),
+                    armDataManagementConfiguration.getTempBlobWorkspace(),
+                    appendUuidToWorkspace
+                );
+
+                if (nonNull(jsonPath) && jsonPath.toFile().exists()) {
+                    ArmResponseCreateRecord armResponseCreateRecord = objectMapper.readValue(jsonPath.toFile(), ArmResponseCreateRecord.class);
+                    UploadNewFileRecord uploadNewFileRecord = readInputJson(armResponseCreateRecord.getInput());
+                    if (nonNull(uploadNewFileRecord)) {
+                        armBatchResponses.addResponseBatchData(Integer.valueOf(uploadNewFileRecord.getRelationId()),
+                                                               armResponseCreateRecord, createRecordFilenameProcessor);
+                    } else {
+                        log.warn("Failed to obtain relation id from create record");
+                    }
+                } else {
+                    log.warn("Failed to write create record file to temp workspace {}", createRecordFilenameProcessor.getCreateRecordFilename());
+                }
+            } catch (IOException e) {
+                log.error("Unable to write create record file to temporary workspace {} - {}", createRecordFilenameProcessor.getCreateRecordFilename(),
+                          e.getMessage());
+            } catch (Exception e) {
+                log.error("Unable to process arm response create record file {} - {}", createRecordFilenameProcessor.getCreateRecordFilename(), e.getMessage());
```

Comment:
e.getMessage() or e ?

Reply:
Full stack trace isn't required on the IO exception but could be used on general exception